src/worker.rs

```diff
@@ -153,7 +153,7 @@ async fn start_worker(server_addr: String) {
                                     Ok(i) => Ok(i),
                                 };
 
-                                if let Err(err) = tx.send(r).await {
+                                if let Err(err) = tx.try_send(r) {
```

Comment:
The freeze problem is solved, but I got another problem: `ERROR shutdown: skywalking_agent::channel: channel send failed err=Broken pipe (os error 32)`, I want to solve it together.